Sources/RealModule/Angle.swift

```diff
@@ -116,6 +116,50 @@ public extension Angle {
     static func atan2(y: T, x: T) -> Self { Angle.radians(T.atan2(y: y, x: x)) }
 }
 
+extension Angle: AdditiveArithmetic {
+    public static var zero: Angle<T> { .radians(0) }
+    
+    public static func + (lhs: Angle<T>, rhs: Angle<T>) -> Angle<T> {
+        Angle(radians: lhs.radians + rhs.radians)
```

Comment:
This seems unfortunate to me that adding two angles specified in degrees incurs rounding error _seven_ times if I want the result in degrees. I think this dedicated type needs to store values given in degrees as-is if it offers such arithmetic.

Reply:
This what I intended with my earlier comment, so degrees and radians are stored separately.
https://github.com/apple/swift-numerics/pull/169#issuecomment-742112265

We could then switch on all operations performed on the `input` and essentially handle three cases:

- degrees with degrees -> perform operation in degrees
- radians with radians -> perform operation in radians
- mixed units -> perform in a common unit

Reply:
👍 That would yield a superior result in the case of operations performed entirely in degrees or radians.

I wonder if there are alternative designs that can improve the result when working with both degrees and radians. If the type stored _both_ `radians` and `degrees`, for example, then you could store the result of "90 degrees plus 1 radian" exactly. I haven't thought through the remainder of the design, but I offer it for your consideration here.